### PR TITLE
WIP: Upgrade packages

### PR DESCRIPTION
**Pull request by igorshapiro**
Add bdd-lazy-var to tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+# 2.0.0 (2018-10-26)
+- deps: upgrade packages
+- Add **bdd-lazy-var** to tests
+- Add **Q.app.isXXXEnv** (e.g. `Q.app.isTestEnv` instead of `Q.app.env === 'test'`)
+- Plugins: Enhance loading (support for non-local plugins)
+- Plugins: Support conditional plugins
+- Add `@aliases` module property support
+- Remove support for generator functions (in DI container)
+- Move test framework (and dependenciers) to `quadro-test` plugin
+- Do not run jobs in test env and in repl modes
```
Comment (tomeresk) [inline CHANGELOG.md]: We should probably also do this for `tasks`

Also, repl mode had an additional change to not run the web server which is not listed here (and we probably should list it).
Not running the web server for tasks as well also seems like a good idea.

Comment (igorshapiro) [reply CHANGELOG.md]: @tomeresk Done

File: lib/jobs/jobs.js [hunk 2]
```diff
@@ -24,6 +27,12 @@ module.exports = class {
       .then(() => this.scheduleJob(job, name))
   }
 
+  shouldSchedule(name) {
+    return this.config.get(`quadro.jobs.${name}.enabled`,
+      this.config.get(`quadro.jobs.all.enabled`, !this.app.isTestEnv && !this.app.isREPL)
```
Comment (tomeresk) [inline lib/jobs/jobs.js]: We should probably also add `!this.app.isTask` here

**Comment by ankurjain86** [comment]
@ishapiro-lemonade should we create a separate org for Quadro and move all the plugins to that place.
Also, there might be some administrative tasks like adding 
1. Create a new org
2. Move repositories to new org
3. Add/Update License to each of these plugins
4. Setting up a build for each of them
5. Update the committers list
6. Setting up slack/project board for coordination and communication.

**Comment by dkvasnicka** [comment]
Too big a PR to reconcile and merge at this point. Once we start devoting more time to evolving Quadro we'll get back to the ideas in this branch and incorporate them if/as necessary.

**Pull request closed without merging**